string camera_name
---
int16 success
int16 pid
